plugins/examples/script-expert-scoring/src/yamlRestTest/resources/rest-api-spec/test/script_expert_scoring/20_score.yml

```diff
@@ -4,33 +4,34 @@
 setup:
   - do:
       indices.create:
-          index:  test
+        index: test
 
   - do:
       index:
-        index:  test
-        id:     "1"
-        body:   { "important_field": "foo" }
+        index: test
+        id: "1"
+        body: { "important_field": "foo" }
   - do:
-        index:
-          index:  test
-          id:     "2"
-          body:   { "important_field": "foo foo foo" }
+      index:
+        index: test
+        id: "2"
+        body: { "important_field": "foo foo foo" }
   - do:
-        index:
-          index:  test
-          id:     "3"
-          body:   { "important_field": "foo foo" }
+      index:
+        index: test
+        id: "3"
+        body: { "important_field": "foo foo" }
 
   - do:
-      indices.refresh: {}
+      indices.refresh: { }
 ---
 "document scoring":
   - do:
       search:
         rest_total_hits_as_int: true
         index: test
         body:
+          explain: true
```

Comment:
Sorry for not mentioning this before, but we should have two tests here. One with explain and the other without.

The one with explain & your new assertion will likely need to skip old cluster versions (cluster_features: gte_v8.16.0` (or something to that effect).). 

Then when backport to 8.15.1, that skip logic will need to be updated to account for 8.15.1.

Then once the backport is merged, main will need to be updated so that it tests against 8.15.1 as well.

Reply:
makes sense; i'll update the test and track this as it's backported.  Thanks!